src/admin.rs

```diff
@@ -211,6 +213,24 @@ pub fn load_azure() -> Result<Arc<dyn ObjectStore>, Box<dyn Error>> {
     Ok(Arc::new(builder.build()?) as Arc<dyn ObjectStore>)
 }
 
+/// Creates a checkpoint of the db stored in the object store at the specified path using the
+/// provided options. Note that the scope option does not impact the behaviour of this method.
+/// The checkpoint will reference the current active manifest of the db.
```

Comment:
now I'm thinking it would be cleaner to just pass the scope as a separate parameter from the `options`. That way we can just exclude it from this fn and only pass it to `Db::create_checkpoint`

Reply:
+1. I had the same thought.